eli5/sklearn/text.py

```diff
@@ -1,8 +1,11 @@
 from __future__ import absolute_import
 from typing import Any, Union, Callable, Dict, List, Optional, Set, Tuple
 
-from sklearn.feature_extraction.text import VectorizerMixin  # type: ignore
 from sklearn.pipeline import FeatureUnion  # type: ignore
+try:
+    from sklearn.feature_extraction.text import _VectorizerMixin as VectorizerMixin
+except ImportError:  # Changed in scikit-learn 0.22
```

Comment:
FWIW it still works in sklearn 0.22, but it's already deprecated (to be removed in 0.24), so good that we are ready 👍